### PR TITLE
Use the The 3-Clause BSD License from opensource.org

### LINKED ISSUE #11009: Revisit LICENSE.md?
The current [LICENSE.md](https://github.com/w3c/web-platform-tests/blob/master/LICENSE.md) comes from history in W3C around test suites and hasn't been looked at in the past 4 years at least.

We should look at it again and see if it still makes sense. If not, we need to figure out what would make sense. and then a transition path from the current one to a new one.

### PR DESCRIPTION
**Pull request by annevk**
Fixes #11009.

### CODE REVIEW

**Comment by annevk** [comment]
@foolip @plehegar @wseltzer I created this to more easily see the differences you pointed out. Subtle modifications seem suboptimal to me. I'd rather we use the original text as-is.

We should also consider adding CODEOWNERS for LICENSE.md.

File: LICENSE.md
Comment (annevk) [inline LICENSE.md]: This is the biggest change here. I wonder if this works with web-platform-tests not being a legal entity.

**Comment by wseltzer** [comment]
I propose we stick with the BSD variant that was already part of the dual license, as it's simpler just to remove one part of a dual license and preserve the attribution clauses some think important. 

In any event we'll need to review the proposed change with the PSIG (Patents and Standards Interest Group), so I'll share a pointer to this thread.

**Comment by annevk** [comment]
@wseltzer that variant doesn't have the copyright line and refers to the W3C twice, which seems weird if this is meant to be BSD. Are those considered identical enough for projects to use it without issue?

**Comment by plehegar** [comment]
One potential issue with the W3C 3-clause license is that it restricts the prevention of endorsements to "W3C nor the names of its contributors". Since WHATWG chose to use WPT, is it also protected from being used to endorse or promote products derived from WPT without their specific prior written permission? Certainly, we'd want all organizations choosing WPT to be protected (unless we prefer to rely on social pressure?). However, I don't think that the boilerplate BSD 3-clause license would fix that.

**Comment by vfournier17** [comment]
@wseltzer This is the issue raised at PSIG yesterday, correct?

**Comment by foolip** [comment]
> One potential issue with the W3C 3-clause license is that it restricts the prevention of endorsements to "W3C nor the names of its contributors".

Would it be possible to say "web-platform-tests project nor the names of its contributors"? Given that the W3C is one of the contributors, I think that'd actually be the same set of entities?

**Comment by annevk** [comment]
You shouldn't remove the documentation link from CONTRIBUTING.md I think.

**Comment by foolip** [comment]
> You shouldn't remove the documentation link from CONTRIBUTING.md I think.

Restored in https://github.com/web-platform-tests/wpt/pull/11191/commits/d9c56ebfa3c44f224671da5b3040c27123f5a97f

**Review by foolip — APPROVED** [review]
LGTM but someone should review the final commit (d9c56ebfa3c44f224671da5b3040c27123f5a97f) which I made.

**Comment by foolip** [comment]
@annevk has reviewed commit d9c56ebfa3c44f224671da5b3040c27123f5a97f, discussed on IRC.

**Comment by foolip** [comment]
Pending Taskcluster status is because this branch is old, before .taskcluster.yml had any jobs defined for PRs. I don't think lint will be affected by this, but I'll rebase the branch to make sure.

**Comment by foolip** [comment]
@annevk suggested squash and merge, and I'd use this commit message:
```
Use the The 3-Clause BSD License from opensource.org (#11191)

Matches https://github.com/web-platform-tests/wpt/pull/13650 except for
trivial differences, namely "The" in title and whitespace.

Also simplify CONTRIBUTING.md to just reference license and docs.

Fixes #11009.
```

**Comment by foolip** [comment]
To repeat what the commit message says, this was effectively the same as https://github.com/web-platform-tests/wpt/pull/13650. It would have been nice to preserve everything in the commit history by merging both PRs, but that would have been messier to follow. I'll close https://github.com/web-platform-tests/wpt/pull/13650, but it's effectively been merged via this.

**Pull request merged**
